lambdas/handlers/create_document_reference_handler.py

```diff
@@ -20,27 +20,25 @@
 def lambda_handler(event, context):
     logger.info("Starting document reference creation process")
 
-    try:
-        document_type = (event["queryStringParameters"]["documentType"]).upper()
-        if document_type in SupportedDocumentTypes.list_names():
-            logger.info("Provided document is supported")
-        else:
-            raise Exception("Provided document type is not supported")   
-    except Exception as e:
+    document_type_string = (event["queryStringParameters"]["documentType"]).upper()
+    document_type = SupportedDocumentTypes.get_from_field_name(document_type_string);
+    if document_type is None:
         logger.error(e)
```

Comment:
There's no error event called e anymore. Please remove this and log out something otherwise we will get errors here